dspace-api/src/main/java/cz/cuni/mff/ufal/dspace/discovery/SolrServiceTweaksPlugin.java

```diff
@@ -45,14 +45,15 @@ public void additionalSearchParameters(Context context,
     {
         String query = discoveryQuery.getQuery();
         // follows previous impl, but is it correct?
-        if (query == null)
-        {
+        if (query == null) {
             query = "*:*";
+        } else {
+        	query = "+(" + query + ")"
```

Comment:
which solr query parser is being used?
comment on what was tried to be achieved here